Tests/LegacyUnit/FrontEnd/EventEditorTest.php

```diff
@@ -1850,6 +1850,101 @@ public function sendEMailToReviewerUsesFrontEndUserNameAsFromNameForMail()
             ]
         );
 
+        $GLOBALS['TYPO3_CONF_VARS']['MAIL']['defaultMailFromAddress'] = 'system-foo@example.com';
+        $GLOBALS['TYPO3_CONF_VARS']['MAIL']['defaultMailFromName'] = 'Mr. Default';
+
+        $this->subject->sendEMailToReviewer();
+
+        self::assertContains(
+            'Mr. Default',
```

Comment:
Please let's move this to a local variable to make the relation more explicit.

Reply:
Local variable or maybe something like this?: 
```
self::assertContains(
    $GLOBALS['TYPO3_CONF_VARS']['MAIL']['defaultMailFromName'],
    $this->mailer->getFirstSentEmail()->getFrom()
);
```

Reply:
A local variable, please. This will minimize the cognitive overhead for reading and understanding the code.